solutions/usaco-839.mdx

```diff
@@ -0,0 +1,48 @@
+---
+id: usaco-839
+source: USACO Gold 2018 Open
+title: Talent Show
+author: Arpan Banerjee
+---
+
+## Incorrect Solution — Common Misconception/Fakesolve
```

Comment:
just write knapsack solution or something the other words are unnecessary, also you should transfer over the official code and call it binary search solution or something